src/tests.builds

```diff
@@ -25,6 +25,13 @@
   <Import Project="$(ToolsDir)tests.targets" Condition="Exists('$(ToolsDir)tests.targets')" />
   <Import Project="$(ToolsDir)VersionTools.targets" Condition="Exists('$(ToolsDir)VersionTools.targets')" />
 
+  <Target Name="BinPlaceXUnitRuntimeForNetstandardSuite" Condition="'$(TargetGroup)' == 'netstandard'" BeforeTargets="BuildAllProjects">
```

Comment:
I don't love the fact of having to do this, but given that our test assemblies have `ReferenceFromRuntime`(for xunit*.dll) and we are building the NETStandard vertical then we need the xunit assemblies to get binplaced there as well. One other option would be to binplace them there at the time we are building `external\dir.proj`. Thoughts?

Reply:
It seems like a better approach might be to have some sort of override such that we build and run on a concrete framework but we force it to pick the netstandard configuration for the test build if it is present.

Reply:
The xunit assemblies that are picked here can't be truly netstandard assemblies as xunit doesn't support these. xunit.console.exe/dll, xunit.execution.dll and others will always be framework specific. There are only a few xunit assemblies which are real netstandard ones. How should this work, i.e. when using xunit.console.dll which requires a host?

Reply:
Even though we are adding a reference to all, we are only really adding a reference to Xunit.NETCore.Extensions, xunit.core, and xunit.assert. Those are the only 3 we use to actually compile test assemblies. The purpose of this PR isn't to produce a runtime in order to run the actual tests using the netstandard configurations, but instead to simply build the netsatndard test suite containing only our test libraries built against netstandard so that we can share that with other frameworks that want to verify that they pass our functionality requirements.